CHANGES.rst

```diff
@@ -6,6 +6,12 @@ There's a frood who really knows where his towel is.
 2.3 (unreleased)
 ^^^^^^^^^^^^^^^^
 
+- Flake8 cleanup
```

Comment:
IMO, changelog entries should provide useful information for end users, developers and integrators; Flake8 fixes doesn't fall under this... but that's only MO.